<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>land</class>
 <widget class="QMainWindow" name="land">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>land</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>480</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>682</width>
       <height>402</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget_image" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>400</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>400</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_login">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(76,73,196);
border-top-right-radius:30px;
border-bottom-right-radius:30px;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="labelforget">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>260</y>
           <width>71</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <underline>true</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(89, 166, 244);
border:0px;
background-color: rgba(255, 255, 255,0);</string>
         </property>
         <property name="text">
          <string>忘记密码？</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="checkBox">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>260</y>
           <width>92</width>
           <height>23</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(102,103,96,0.5);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>记住密码</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_login">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>40</y>
           <width>151</width>
           <height>61</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>70</horstretch>
           <verstretch>70</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Ubuntu Condensed</family>
           <pointsize>40</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(255, 255, 255,0);
color: rgb(238, 238, 236);</string>
         </property>
         <property name="text">
          <string>LOGIN</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_password">
         <property name="geometry">
          <rect>
           <x>21</x>
           <y>204</y>
           <width>260</width>
           <height>45</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
    border: 1px solid #33465d;
    border-radius: 10px</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_15">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>11</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_password">
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 17pt &quot;Ubuntu Condensed&quot;;
border-image: url(:/icon/image/iocn/password.png);
border:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_19">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>11</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLineEdit" name="passwordLineEdit">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px</string>
            </property>
            <property name="placeholderText">
             <string>请输入你的密码</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_13">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>11</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frame_id">
         <property name="geometry">
          <rect>
           <x>21</x>
           <y>152</y>
           <width>260</width>
           <height>45</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>260</width>
           <height>45</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:white;
    border: 1px solid #33465d;
    border-radius: 10px</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>11</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_id">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>35</horstretch>
              <verstretch>35</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Ubuntu Condensed</family>
              <pointsize>17</pointsize>
              <weight>9</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 17pt &quot;Ubuntu Condensed&quot;;
border-image: url(:/icon/image/iocn/user.png);
border:none;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_18">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>11</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="userComboBox">
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border:0px;
background-color: rgb(255, 255, 255);
QComboBox::drop-down { border:0px;border-left-width: 0px;};
QComboBox::down-arrow {border:0px;image: url(:/icon/image/iocn/down.png);};
QComboBox::down-arrow:on {border:0px;image: url(:/icon/image/iocn/down.png);}
</string>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_17">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>11</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>40</x>
           <y>290</y>
           <width>241</width>
           <height>42</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_login_p">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>60</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(238, 238, 236);
border-radius:10px</string>
            </property>
            <property name="text">
             <string>人脸识别</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_login">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>60</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>40</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>40</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(238, 238, 236);
border-radius:10px</string>
            </property>
            <property name="text">
             <string>登陆</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_background">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>800</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_background</zorder>
    <zorder>layoutWidget</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
